source/_integrations/lektrico.markdown

```diff
@@ -0,0 +1,79 @@
+| state               |      | State of the charger.                                     |
+| charging_time       | s    | Indicates the current session charging time.              |
+| power               | kW   | Current instant power.                                    |
+| energy              | kWh  | Current session charged energy.                           |
+| temperature         | °C   | Board temperature.                                        |
+| lifetime_energy     | kWh  | Total charged energy since installation.                  |
+| installation_current| A    | Current value [A] to be limited by software.              |
+| limit_reason        |      | Current limit reason.                                     |
+| voltage             | V    | Measured voltage.                                         |
+| current             | A    | Measured current.                                         |
+
+### Three-phase charger
+
+| Condition           | Unit | Description                                               |
+| :------------------ | :--- | :-------------------------------------------------------- |
+| state               |      | State of the charger.                                     |
+| charging_time       | s    | Indicates the current session charging time.              |
+| power               | kW   | Current instant power.                                    |
+| energy              | kWh  | Current session charged energy.                           |
+| temperature         | °C   | Board temperature.                                        |
+| lifetime_energy     | kWh  | Total charged energy since installation.                  |
+| installation_current| A    | Current value [A] to be limited by software.              |
+| limit_reason        |      | Current limit reason.                                     |
+| voltage_l1          | V    | Measured voltage on L1.                                   |
+| voltage_l2          | V    | Measured voltage on L2.                                   |
+| voltage_l3          | V    | Measured voltage on L3.                                   |
+| current_l1          | A    | Measured current on L1.                                   |
+| current_l2          | A    | Measured current on L2.                                   |
+| current_l3          | A    | Measured current on L3.                                   |
+
+### Single-phase energy meter
+
+| Condition           | Unit | Description                                               |
+| :------------------ | :--- | :-------------------------------------------------------- |
+| breaker_current     | A    | Main breaker current.                                     |
+| power               | kW   | Measured active power.                                    |
+| pf                  |      | Power factor.                                             |
```

Comment:
what is a power factor?

Reply:
It's a common value in electricity, it measures the ratio of active power to apparent power.
It's also a Home Assistant sensor device class (SensorDeviceClass.POWER_FACTOR).